CHANGES_NEXT_RELEASE

```diff
@@ -2,3 +2,5 @@
 - Add: Allow notices and rules paths to be configured by environment variables (issue #161)
 - Fix: align and fix log traces, add comp field (issue #166)
 - Add: add 'from' field in log traces (issue #166)
+- Fix: env var for next core bug (issue #171)

```

Comment:
Already described in https://github.com/telefonicaid/perseo-fe/blob/develop/documentation/configuration.md#environment-variables-configuration

NTC